tests/suites/test_suite_constant_time.function

```diff
@@ -35,17 +35,25 @@ void mbedtls_ct_memcmp(int same, int size, int offset)
     TEST_CF_SECRET(a + offset, size);
     TEST_CF_SECRET(b + offset, size);
 
+    /* Construct data that matches, if same == -1, otherwise
```

Comment:
So it's not really (boolean) `same` but `failing_offset` now?

Reply:
Yes. I guess renaming it would be nicer, but I'll hold off unless there is more to do.